test/e2e/test-apps/other/custom-tracing/src/main.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line import/order
+const { app } = require('electron');
+
+const Sentry = require('@sentry/electron');
+// eslint-disable-next-line no-unused-vars
+const Tracing = require('@sentry/tracing');
```

Comment:
A simple require for the side effect should be enough?

```suggestion
require('@sentry/tracing');
```

Reply:
I've just tested it and it works without `@sentry/tracing` entirely 🤔

Reply:
🤔 Are we doing the require call somewhere else?

Reply:
No!  🤣

Could it be this?
https://github.com/getsentry/sentry-javascript/blob/6809dd2ae41f81e3b5d3e3c8076f2c6bcb9cfbfb/packages/node/src/handlers.ts#L4

Reply:
It’s the first one! Lol